lib/assets/client.js

```diff
@@ -41,6 +41,12 @@ function invite(channel, coc, email, fn){
     email: email
   })
   .end(function(res){
+    if (res.body.redirectUrl) {
+      var err = new Error(res.body.msg || 'Server error');
+      window.setTimeout(() => {
+        location.href = res.body.redirectUrl;
+      }, 1500);
```

Comment:
@MaxWofford @rauchg Do you feel strongly about this timeout? I kinda hate it. If you're fond of it, I can try to write a PR to add a `redirectDelay` option or whatever, but if you're ambivalent about this I just want to delete it?